elliptic-curve/src/group_digest.rs

```diff
@@ -0,0 +1,86 @@
+use crate::hash2field::{hash_to_field, ExpandMsg, FromOkm};
+use crate::isogeny::Isogeny;
+use crate::osswu::OsswuMap;
+use ff::Field;
+use group::Group;
+
+/// Convert a set of field elements into a point
+pub trait ConvertToPoint {
+    /// The field element that represents a point
+    type FieldElement: Field;
+    /// The result of converting two field elements to a Point
+    type Output;
+
+    /// Converts two coordinates into a point
+    fn convert_to_point(x: Self::FieldElement, y: Self::FieldElement) -> Self::Output;
+}
```

Comment:
I think there is a missing trait `MapToCurve` that should replace this. It should represent `map_to_curve` from the spec.

Reply:
It's clear to me that this naming doesn't convey what I want it to and that is we need a trait for converting two field elements into a point. MapToCurve is a separate method which I will add as I think that's a great suggestion. However, this trait is necessary since we don't have a method anywhere like From or anything else that allows passing in two field elements for x and y and returns the point. This is available directly with specific curves like `p256` and `k256` where one can use AffinePoint { x, y, infinity: false } but outside of that its not currently possible.